.github/workflows/build.yml

```diff
@@ -41,9 +41,9 @@ jobs:
         uses: dtolnay/rust-toolchain@stable
 
       - name: Cache Rust dependencies
-        uses: Swatinem/rust-cache@98c8021b550208e191a6a3145459bfc9fb29c4c0
+        uses: Swatinem/rust-cache@v2
```

Comment:
Switched to using version tags (`@v2`) instead of commit hashes - cleaner and more standard practice

Reply:
Codacy might complain about unpinned versions, but that's fine - these actions are safe and we want automatic patch updates anyway ：）